### PR TITLE
Manifest documentation

### PR DESCRIPTION
**Pull request by dwapstra**

### CODE REVIEW

File: docs/manifest/introduction.rst [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    $ pyats run manifest job.tem --profile sanity
+
+Command line manifest validation:
+
+.. code-block:: shell
+
+    $ pyats validate manifest job.tem
+
+The manifest defines the script execution arguments, runtime environment and the profile(s)
+to define environment specific settings and arguments. The profiles are intended to allow
```
Comment (simingy) [inline docs/manifest/introduction.rst:25]: the profiles _that_?

Comment (simingy) [reply docs/manifest/introduction.rst:25]: that define environment specific settings and arguments. Profiles allow

File: docs/manifest/introduction.rst [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+the same script to be run against mulitple environments, e.g. local testbed, orchestrated
```
Comment (simingy) [inline docs/manifest/introduction.rst:26]: , and or run with different input parameters,

Comment (simingy) [reply docs/manifest/introduction.rst:26]: eg, for different scaling numbers, purposes

File: docs/manifest/introduction.rst [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+.. code-block:: yaml
+
+    arguments:
+        mail-html: True
+        configuration: easypy_config.yaml
+
+    profile:
+        local:
+            arguments:
```
Comment (simingy) [inline docs/manifest/introduction.rst:53]: i think here it's good convention to produce a set of rules of how these yaml sections gets converted into your cli arguments

one that is apparent is anything will get prefixed with --.
what else?

File: docs/manifest/schema.rst [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+           # required
+
+    arguments:  # arguments for the script (optional)
+
+        # command line arguments may override the arguments provided here or in the profile
+
+        <key>: <value>  # if the key does not start with "-", "--" will be prepended to the argument string, e.g. --key value
+
+        # argparse 'nargs' using 'list'
+        key:  # values may be specified as a list, this will translate to "--key val1 val2"
```
Comment (simingy) [inline docs/manifest/schema.rst:20]: i think this schema will be somewhat more difficult to read.

it contains a lot of critical information

maybe worth reformatting and or building a quick reference/rule set

File: docs/manifest/introduction.rst [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+these.
+
+For example running the manifest execution with the above arguments and adding the
+`testbed-file` argument on the command line will override the profile argument.
+
+.. code-block:: shell
+
+    $ pyats validate manifest job.tem --profile local --testbed-file testbed2.yaml
+
+
```
Comment (simingy) [inline docs/manifest/introduction.rst:67]: i think you missed out the part where a tem file is associated with a job file - what's the association rule

**Review by simingy — CHANGES_REQUESTED** [review]

**Comment by dwapstra** [comment]
@simingy  thanks for the feedback, I have updated the documentation. Please review.

**Review by simingy — APPROVED** [review]

**Review by LukasMcClelland — APPROVED** [review]

**Pull request merged**